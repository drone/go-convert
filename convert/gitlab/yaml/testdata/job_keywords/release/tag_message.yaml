# https://docs.gitlab.com/ee/ci/yaml/#releasetag_message

  release_job:
    stage: release
    release:
      tag_name: $CI_COMMIT_TAG
      description: 'Release description'
      tag_message: 'Annotated tag message'