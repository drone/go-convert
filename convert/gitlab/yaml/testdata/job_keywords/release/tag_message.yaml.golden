release_job:
    release:
        tag_name: $CI_COMMIT_TAG
        tag_message: Annotated tag message
        description: Release description
    stage: release
